src/library_glfw.js

```diff
@@ -737,7 +741,6 @@ var LibraryGLFW = {
 
       event.preventDefault();
 
-#if '$FS' in addedLibraryItems
```

Comment:
But we still use `FS` unconditionally below?  Shouldn't we wrap this somehow in `if FILESYSTEM` too?  Maybe I'm missing something?

Reply:
Might as well ifdef it, yeah, thanks. Done now.

In practice probably an app wouldn't have drag&drop without filesystem support, but ifdefing is safer (and also smaller).